### DISCUSSION TITLE
failing pr-pull request using github action

### DISCUSSION THREAD

**Discussion by ipatch** [Homebrew/discussions#2030] [Tap maintenance and Homebrew development] [answered]
i successfully created bottles for

- mojave
- catalina
- big sur

using brew test-bot and the ci completed with a green check mark ✅

so i label the PR request as `pr-pull` to begin the publish action / workflow, but am seeing the below error,

https://github.com/FreeCAD/homebrew-freecad/pull/209/checks?check_run_id=3396377268

how do i resolve this issue?

**Comment by SMillerDev** [comment, marked as answer]
It needs to actually update something to be able to pull and publish an update. You can add `revision 1` to the formula and it'll likely go through.
